react-file-uploads/client/src/UploadForm.js

```diff
@@ -21,15 +21,16 @@ export default  function UploadForm() {
      */
     event.preventDefault();
     const formData = new FormData(event.target);
-
     const res = fetch('http://localhost:8080/api/uploads', {
       method: "POST",
       body: formData,
   });
-    const jsonData = await res;
-
+    if (!res.ok) {
+      throw new Error(`error ${res.status}`);
```

Comment:
Almost. This is good now, but you need to use a try/catch to catch exceptions.

Reply:
oh, I wasn't done, I just pushed because I had to restart VS code and didn't want to lose anything